x-pack/plugins/observability_solution/observability_ai_assistant_app/server/rule_connector/index.ts

```diff
@@ -37,14 +36,22 @@ import { CompatibleJSONSchema } from '@kbn/observability-ai-assistant-plugin/com
 import { AlertDetailsContextualInsightsService } from '@kbn/observability-plugin/server/services';
 import { getSystemMessageFromInstructions } from '@kbn/observability-ai-assistant-plugin/server/service/util/get_system_message_from_instructions';
 import { AdHocInstruction } from '@kbn/observability-ai-assistant-plugin/common/types';
+import { EXECUTE_CONNECTOR_FUNCTION_NAME } from '@kbn/observability-ai-assistant-plugin/server/functions/execute_connector';
 import { convertSchemaToOpenApi } from './convert_schema_to_open_api';
 import { OBSERVABILITY_AI_ASSISTANT_CONNECTOR_ID } from '../../common/rule_connector';
 
 const CONNECTOR_PRIVILEGES = ['api:observabilityAIAssistant', 'app:observabilityAIAssistant'];
 
+export const SlackConnectorParamsSchema = schema.object({
+  id: schema.string({ minLength: 1 }),
+  params: schema.object({
+    message: schema.string({ minLength: 1 }),
+  }),
+});
+
 const connectorParamsSchemas: Record<string, CompatibleJSONSchema> = {
   '.slack_api': convertSchemaToOpenApi(SlackApiParamsSchema),
-  '.slack': convertSchemaToOpenApi(SlackParamsSchema),
+  '.slack': convertSchemaToOpenApi(SlackConnectorParamsSchema),
```

Comment:
you can just inline the JSON schema here btw, no need to convert from kbn/config-schema.